### DISCUSSION TITLE
Deezer (not) matching

### DISCUSSION THREAD

**Discussion by burnshroom** [RandomNinjaAtk/arr-scripts#115] [Q&A] [answered]
I have seen other mentioning this issue and saw a response regarding issue #67 possibly being behind the issue.

By inputting the arl token, the script does not use backup search method, Freyr so once it completes it's search via Deezer the script idles until it's restart time limit is reached.  Using the script as is and by simply setting up one artist (Taylor Swift) it searchs for 158 items and finds none.

I can go to the actual website and see titles it is searching for, but the script cannot find them via it's search methods.

Viewing the extended.conf file I tried changing the matchDistance from the default "5" to a higher number to increase the threshold and as a result of moving it to 15, it found one result and proceeded to not find anymore.  

 I tried increasing the matchDistance several times and had just a bit more success.  Changing it to 30, 100, 200 and finally 999 (and each time deleting the "notfound" folder) it ended up locating 8 different items and not finding the other 150 items.

Here are several logs showing the searches and results with the changes I made from above.

[Audio-2023_11_18_01_41_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399617/Audio-2023_11_18_01_41_AM.txt)
[Audio-2023_11_18_01_58_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399618/Audio-2023_11_18_01_58_AM.txt)
[Audio-2023_11_18_02_07_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399619/Audio-2023_11_18_02_07_AM.txt)
[Audio-2023_11_18_02_11_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399620/Audio-2023_11_18_02_11_AM.txt)
[Audio-2023_11_18_02_20_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399621/Audio-2023_11_18_02_20_AM.txt)
[Audio-2023_11_18_02_24_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399622/Audio-2023_11_18_02_24_AM.txt)
[Audio-2023_11_18_02_27_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399623/Audio-2023_11_18_02_27_AM.txt)
[Audio-2023_11_18_02_30_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399624/Audio-2023_11_18_02_30_AM.txt)
[Audio-2023_11_18_02_33_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399625/Audio-2023_11_18_02_33_AM.txt)
[Audio-2023_11_18_02_36_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399626/Audio-2023_11_18_02_36_AM.txt)
[Audio-2023_11_18_02_39_AM.txt](https://github.com/RandomNinjaAtk/arr-scripts/files/13399627/Audio-2023_11_18_02_39_AM.txt)

I unfortunately don't have experience debugging code to the point where I can see what is going on.  I would love to see what exactly is being matched in the searches.  

Hoping you might see something based on the logs.

**Comment by RandomNinjaAtk** [comment, marked as answer]
Latest updates I think provides a bit more logging enhancements... Update and see if it's any better..
